### DISCUSSION TITLE
First Time Connection Editing

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#220] [General]
(Closed) Issue #214 seems to have been solved by the user, but it looks like there might be a bug that occurs if the user has no home directory.  I think that makes two first time users in the last two weeks who have **"unusual"** user profiles.  So it might bear looking into--maybe provide something like you did with tempLibrary.  
(Not sure if this is the right place to bring this up. Please delete if inappropriate.)

**Comment by worksofliam** [comment]
I saw that too. I am going to investigate tonight. I believe I know the issue.

**Comment by Dman247** [comment]
I think #214 was dealing with the temporary library. 
I believe you're referencing this issue https://github.com/halcyon-tech/vscode-ibmi/issues/218#issuecomment-883652072

**Comment by worksofliam** [comment]
Create issue #221
